deploy/catalog_resources/community/kiecloud-operator.v1.0.0.clusterserviceversion.yaml

```diff
@@ -39,7 +39,7 @@ spec:
     - name: Product Page
       url: https://access.redhat.com/products/red-hat-process-automation-manager
     - name: Documentation
-      url: https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.2/html-single/installing_and_configuring_red_hat_process_automation_manager_on_red_hat_jboss_eap_7.2/
+      url: https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.3/html/deploying_a_red_hat_process_automation_manager_authoring_environment_on_red_hat_openshift_container_platform/
```

Comment:
Why authoring? Maybe this instead, changing 7.2 to 7.3?

https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.2/#category-deploying-red-hat-process-automation-manager-on-openshift

Reply:
good point, changed to 
https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.3/#category-deploying-red-hat-process-automation-manager-on-openshift

Reply:
Thanks, is there a staging area for docs where you can verify this link?